taipy/gui/utils/get_module_name.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import inspect
+import sys
+import typing as t
+from types import FrameType, ModuleType
+
+
+def _get_module_name_from_frame(frame: FrameType):
```

Comment:
Since there are 2 get_module_name function, I want to add this prefix so that there would be no confusion. It is ashamed that python doesnt support the same function with different parameter like Java and C++ but it is to be expected since there is the kwargs aspect to python function.